pilot/proxy/envoy/testdata/cds-egress-rule-tcp.json.golden

```diff
@@ -0,0 +1,190 @@
+{
+  "clusters": [
```

Comment:
Remove

Reply:
If I remove this, how should I test that clusters are created for egress tcp rule?

Reply:
See earlier comment. Add both egress rules to an existing CDs case. Then you don’t need a dedicated golden